src/types/home.d.ts

```diff
@@ -0,0 +1,12 @@
+interface PanelProps {
+  isActive?: boolean;
+  header?: React.ReactNode;
+  className?: string;
+  style?: React.CSSProperties;
+  showArrow?: boolean;
+  forceRender?: boolean;
+  /** @deprecated Use `collapsible="disabled"` instead */
+  disabled?: boolean;
+  extra?: React.ReactNode;
+  collapsible?: CollapsibleType;
+}
```

Comment:
do you still need this? usually you create type for global to have .d.ts since you are not its better to keep it with the class

Reply:
Removed